style.css

```diff
@@ -272,394 +216,308 @@ a.button {
-  display: flex;
-  align-items: center;
-  margin-bottom: 12px;
+.a1 .pro3a {
+  background-image: url(./images/desktop/image3.png);
+  width: 100%;
+  height: 100%;
+  background-size: 100% 100%;
 }
 
-#about .button-c {
-  width: 327px;
-  height: 40px;
-  display: flex;
-  align-items: center;
-  margin-bottom: 16px;
+.b1 .pro2a {
+  width: 100%;
+  height: 100%;
+  background-size: 100% 100%;
+  background-image: url(./images/desktop/image2.png);
 }
 
-#about .button-c .letsconnect {
-  text-decoration: none;
-  color: var(--color-main);
-  font-size: 11px;
-  font-weight: 500;
+.b1 .pro4a {
+  width: 100%;
+  height: 100%;
+  background-size: 100% 100%;
+  background-image: url(./images/desktop/image4.png);
 }
 
-#about .button-c .letsconnect h2 {
-  font-weight: normal;
+.project-a .a2 {
+  width: 47%;
+  height: 49vh;
+  padding: 24px;
 }
 
-#about .icons {
-  list-style-type: none;
-  padding: 0%;
-  width: 327px;
-  height: 40px;
+#projects .project-b {
+  width: 100%;
+  height: 54vh;
+  background-color: white;
+  border: 1px solid var(--cards-border);
+  border-radius: 24px;
   display: flex;
-  justify-content: flex-start;
-  gap: 10px;
+  justify-content: space-around;
   align-items: center;
-  margin-bottom: 12px;
+  flex-direction: row-reverse;
 }
 
-#about .icons li {
-  display: inline;
+.project-b .b1 {
+  width: 47%;
+  height: 49vh;
 }
 
-#about .icons .icon {
-  width: 25px;
-  height: 25px;
-  margin-right: 6px;
+.project-b .b2 {
+  width: 47%;
+  height: 49vh;
+  padding: 24px;
 }
 
-#about .about-b {
-  width: 327px;
-  height: 464.15px;
-  padding: 15px;
+#about {
+  width: 100%;
+  height: 100.8vh;
+  background-color: white;
+  border-top-right-radius: 180px 180px;
   display: flex;
-  justify-content: space-between;
-  flex-direction: column;
-  align-items: center;
-}
-
-.about-b .frameworks,
-.skills,
-.languages,
-.js {
-  list-style-type: none;
+  flex-direction: row;
+  justify-content: center;
+  gap: 5%;
+  align-items: flex-start;
+  padding-top: 15vh;
 }
 
-#about .about-b .languages {
-  width: 327px;
-  height: 320px;
-  margin-bottom: 30px;
-  display: flex;
-  flex-direction: column;
-  padding: 12px;
+#about .title {
+  font-size: 40px;
+  line-height: 52px;
+  margin-bottom: 12px;
+  color: var(--text-color-titles);
 }
 
-#about .about-b .languages li {
-  list-style-type: none;
+#about .text {
+  font-size: 16px;
+  color: var(--color-regular-text);
+  margin-bottom: 12px;
 }
 
-#about .about-b .languages .lan-button {
-  width: 327px;
-  height: 72px;
-  font-size: 20px;
-  display: table-cell;
-  vertical-align: middle;
-  padding: 0%;
-  line-height: 80px;
+#about .about-a {
+  width: 35%;
+  height: 54vh;
 }
 
-#about .about-b .languages .languages-button .languages-t {
-  width: 250px;
-  height: 72px;
+.pro-button .about-button {
+  text-decoration: none;
+  color: var(--color-main);
+  border: 1px solid var(--color-main);
+  border-radius: 8px;
+  font-size: 17px;
+  width: 124px;
```

Comment:
- [ ] Please adjust the text in this button to match the design given in Figma. It is currently wrapped in a desktop screen.
- [ ] Please add a little spacing of the description text from the top of the `About Myself` heading to match with the Figma design spacing.

Your Design
![image](https://user-images.githubusercontent.com/50941074/160214073-855d95c3-9bfd-474e-8143-0648ae10d045.png)

Figma Design
![image](https://user-images.githubusercontent.com/50941074/160214116-e41be9d0-9d41-41b0-9375-0e87b3e00a93.png)